test/specs/json_ld/conf/google_metadata/schema.json

```diff
@@ -0,0 +1,104 @@
+{
+    "schema_name": "google_metadata_schema",
+    "tables": {
+        "google_metadata_w_handlebars": {
+            "kind": "table",
+            "table_name": "google_metadata_w_handlebars",
+            "schema_name": "google_metadata_schema",
+            "column_definitions": [
+                {
+                    "name": "id",
+                    "nullok": false,
+                    "type": {
+                        "typename": "integer"
+                    }
+                },
+                {
+                    "name": "title",
+                    "nullok": false,
+                    "type": {
+                        "typename": "text"
+                    }
+                },
+                {
+                    "name": "website",
+                    "type": {
+                        "typename": "text"
+                    }
+                },
+                {
+                    "name": "category",
+                    "type": {
+                        "typename": "text"
+                    }
+                },
+                {
+                    "name": "summary",
+                    "type": {
+                        "typename": "text"
+                    }
+                },
+                {
+                    "name": "RMT",
```

Comment:
Remove `RMT` and `RCT` definitions from here. These two are added automatically by test framework.